frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -76,18 +76,16 @@
             <TableOfContents :links="tocLinks" />
             <div id="intro" class="column content has-text-justified">
               <p>
-                GotEnzymes provides open access to over XYZ million predicted kinetic data points
-                for XYZ unique sequences and XYZ compounds from XYZ species with the cutting-edge
-                artificial intelligence tools. The values for kcat [reference] and KM [reference] in
-                this database are predicted from available advanced deep learning tools. Our vision
-                for this database is to store every possible predicted parameter for any available
-                enzyme to facilitate better science.
+                GotEnzymes provides open access to over 25,794,195 million predicted k<sub>cat</sub>
+                for 5,825,213 unique sequences and 4,147 compounds from 8,099 species with the
+                cutting-edge artificial intelligence tools.
               </p>
 
               <hr class="mt-6" />
               <h4 id="how-to-use-it" class="is-info is-size-4">How to use it</h4>
               <p>
-                We currently support searches by reaction, compound, EC number. Each predicted
+                We currently support fuzzy searches by reaction, compound, and EC number, organism
```

Comment:
```suggestion
                We currently support fuzzy searches by reaction, compound,  EC number, organism
```
The `and` before `EC number` should be removed after addition of `organism and domain`

Reply:
Good point @nanjiangshu, I just had to remove the extra space as well
```suggestion
                We currently support fuzzy searches by reaction, compound, EC number, organism
```